site/docs/components/icons.md

```diff
@@ -1,80 +1,58 @@
 ---
 layout: docs
-title: Icons (FloSports) (WIP)
-description: Use SVG Icons in HTML
+title: Icons
+description: SVG Icons in HTML
 group: components
 toc: true
 ---
-# NOT Updated for FLO-SCSS yet
```

Comment:
As a note I think we need to explain the difference between fill="currentColor" and the use of classes in the svg. `class="flo-svg-logo-title"` Why you use one over the other.